vss-extension.json

```diff
@@ -1,6 +1,6 @@
 {
   "manifestVersion": 1,
-  "id": "synopsys-coverity-on-polaris",
+  "id": "synopsys-polaris",
```

Comment:
This change is to related to hotfix, but was used even for the last release of v1.3.0 also. I thought we could change this value, but we cannot change this for an existing extension. And, so reverted this.